dev/BulkLoader.php

```diff
@@ -391,7 +391,7 @@ public function addUpdated($obj, $message = null) {
 	 */
 	public function addDeleted($obj, $message = null) {
 		$data = $obj->toMap();
-		$data['Message'] = $message;
+		$data['_BulkLoaderMessage'] = $message;
```

Comment:
Why did you change Message to _BuklLoaderMessage? It was Message in the original commit; was this a bug?